code/game/objects/items/toys.dm

```diff
@@ -182,6 +183,48 @@
 
 #undef BALLOON_COLORS
 
+/*
+* Captain's Aid
+*/
+#define CAPTAINSAID_MODE_OFF 1
+
+/obj/item/toy/captainsaid
+	name = "\improper Captain's Aid"
+	desc = "Every captain's greatest ally when exploring the vast emptiness of space, now with a color display!"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "captainsaid_off"
+	/// List of modes it can cycle through
+	var/list/modes = list(
+		"off",
+		"port",
+		"starboard",
+		"fore",
+		"aft",
+	)
+	/// Current mode of the item, changed when cycling through modes
+	var/current_mode = CAPTAINSAID_MODE_OFF
+
+/obj/item/toy/captainsaid/examine_more(mob/user)
+	. = ..()
+	. += span_notice("You could swear you've been hearing advertisments for the 'soon upcoming' release of a tablet version for the better part of 3 years...")
+
+/obj/item/toy/captainsaid/attack_self(mob/living/user)
+	current_mode++
+	playsound(src, 'sound/items/screwdriver2.ogg', 50, TRUE)
```

Comment:
```suggestion
	playsound(src, 'sound/items/screwdriver2.ogg', 50, vary = TRUE)
```

Reply:
I don't think any other playsound proc writes it that way, but sure. Improves readability so it's a good precedent.

Reply:
I'm trying to get more people to do it because random `TRUE` isn't readable